flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -729,12 +730,7 @@ private void disposeSavepoint(ClusterClient<?> clusterClient, String savepointPa
 	protected void executeProgram(PackagedProgram program, ClusterClient<?> client, int parallelism) throws ProgramMissingJobException, ProgramInvocationException {
 		logAndSysout("Starting execution of program");
 
-		final JobSubmissionResult result = client.run(program, parallelism);
-
-		if (null == result) {
```

Comment:
never be null

Reply:
What happens if the user program `main()` doesn't call execute? Could it then be `null`?

Reply:
see my comment below. We can always do the null check but it is redundant.